pints/_plot.py

```diff
@@ -0,0 +1,248 @@
+#
+# Quick diagnostic tools for inference results
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+import pints
+import numpy as np
+from scipy import stats
+#import matplotlib.pyplot as plt
```

Comment:
Please use pyplot instead of pylab!

It'd also be good not to import matplotlib in the module, but individually in each method (this doesn't have an impact on performance, the module is still only loaded once).
This way, if people import the plot module Matplotlib isn't automatically imported too (which can help prevent errors on servers etc.)

Reply:
Do you mean having `import matplotlib.pyplot as plt` inside, e.g., `trace()`?

Something like: 
```
def trace(chain):
    """
    ...
    """
    import matplotlib.pyplot as plt
    ...
    return fig
```

And do it for each method?

Reply:
Yeah!